docs/src/components/menu/menu.module.scss

```diff
@@ -0,0 +1,82 @@
+@import '../../pages/global';
+
+.desktopMenu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  width: var(--sidebar-width);
+  overflow-y: auto;
+  @include respond-to('mobile') {
+    display: none;
+  }
+}
+
+.mobileMenu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: var(--topbar-height);
+  display: flex;
+  @include respond-to('desktop') {
+    display: none;
+  }
+}
+
+.mobileMenu,
+.desktopMenu {
+  background-color: $menuBackgroundColor;
+  color: $menuTextColor;
+  padding: 20px;
```

Comment:
Just a thought here. What screen sizes are we supporting? Since it is a desktop app would it drastically change the UX if screen size it twice larger?